plugins/mcreator-themes/themes/matrix/styles/code_editor.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE RSyntaxTheme SYSTEM "theme.dtd">
+
+<RSyntaxTheme version="1.0">
+
+    <background color="323232"/>
+    <caret color="c1cbc2"/>
+    <selection useFG="false" bg="404E51"/>
+    <currentLineHighlight color="2F393C"/>
+    <marginLine fg="394448"/>
+    <markAllHighlight color="6b8189"/>
+    <markOccurrencesHighlight color="5b7179"/>
+    <matchedBracket fg="6A8088" bg="6b8189" highlightBoth="true" animate="true"/>
+    <hyperlinks fg="a082bd"/>
+
+    <secondaryLanguages>
+        <language index="1" bg="334455"/>
+        <language index="2" bg="223322"/>
+        <language index="3" bg="ffe0f0"/>
+    </secondaryLanguages>
+
+    <gutterBorder color="747E82"/>
+    <lineNumbers fg="81969A"/>
+    <foldIndicator fg="6A8088" iconBg="2f383c"/>
+    <iconRowHeader activeLineRange="3399ff"/>
+
+    <tokenStyles>
+        <style token="IDENTIFIER" fg="E0E2E4"/>
+        <style token="RESERVED_WORD" fg="93c54b"/>
+        <style token="RESERVED_WORD_2" fg="93c54b"/>
+        <style token="ANNOTATION" fg="E8E2B7"/>
+        <style token="COMMENT_DOCUMENTATION" fg="6C788C"/>
+        <style token="COMMENT_EOL" fg="66747B"/>
+        <style token="COMMENT_MULTILINE" fg="66747B"/>
+        <style token="COMMENT_KEYWORD" fg="ae9fbf"/>
+        <style token="COMMENT_MARKUP" fg="ae9fbf"/>
+        <style token="FUNCTION" fg="E0E2E4"/>
+        <style token="DATA_TYPE" fg="678CB1" bold="true"/>
+        <style token="LITERAL_BOOLEAN" fg="93c54b"/>
+        <style token="LITERAL_NUMBER_DECIMAL_INT" fg="93c54b"/>
+        <style token="LITERAL_NUMBER_FLOAT" fg="93c54b"/>
+        <style token="LITERAL_NUMBER_HEXADECIMAL" fg="93c54b"/>
```

Comment:
Replace every _93c54b_ with `${mainTint}` token if you want reserved words to be recolored on change of **interfaceAccentColor** (this happened to default themes in #1433).